openshift/pipeline-latest-release.sh

```diff
@@ -14,10 +14,9 @@ function tryurl {
     curl -s -o /dev/null -f ${1} || return 1
 }
 
-if [[ "${1:-}" != "--only-stable-release" ]];then
-  if tryurl ${NIGHTLY_RELEASE};then
+# check for pipeline nightly release if no nightly the go for version based yaml file.
+if tryurl ${NIGHTLY_RELEASE};then
```

Comment:
Won't this be always true? How will it go to `else` statement?

Reply:
Yes else part is just a fallback condition if at anycase nightly release give 404 not found

Reply:
i think in the current scenario, if nightly is not found we should let it fail

Reply:
Yup updated PR